lib/cli.js

```diff
@@ -388,9 +388,16 @@ export default async function loadCli() { // eslint-disable-line complexity
 		exit(error.message);
 	}
 
+	const workerThreads = combined.workerThreads !== false;
+
 	let nodeArguments;
 	try {
 		nodeArguments = normalizeNodeArguments(conf.nodeArguments, argv['node-arguments']);
+		nodeArguments ??= process.execArgv;
```

Comment:
`normalizeNodeArguments` always returns an array, which starts with the `execArgv` values.

```suggestion
```

Reply:
Can't believe I missed that.